packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -1493,6 +1440,53 @@ function workLoopSync() {
   }
 }
 
+function renderRootConcurrent(root, expirationTime) {
```

Comment:
Just to be clear, this is only called once so it didn't technically need to be extracted out, right?

Reply:
Yeah I only did it for symmetry purposes